### ISSUE TITLE
Doorbell Tweets broken

### ISSUE THREAD

**Issue by ikifar2012** [CCOSTAN/Home-AssistantConfig#1032]
It seems it is failing to get the number of doorbell presses per day thus replacing that value with unknown 

Example:
https://twitter.com/BearStoneHA/status/1386108471993638919

**Comment by CCOSTAN** [comment]
Yeah.. Thanks for opening this up.   The doorbell is actually broken.  It's the #259 Skybell doorbell.  Soon to be replaced with the Wyze Doorbell.

**Comment by ikifar2012** [comment]
I thought the Wyze Doorbell had no Home Assistant support. How do you plan on integrating it?

**Comment by CCOSTAN** [comment]
Gonna figure out a way to leverage Alexa Media Player and the Wyze Alexa Skill. 

![image](https://user-images.githubusercontent.com/2160436/116749608-2f310c00-a9cf-11eb-8cf0-3ad531a85081.png)
